dnf5/main.cpp

```diff
@@ -933,6 +933,14 @@ int main(int argc, char * argv[]) try {
     loggers.emplace_back(std::make_unique<libdnf5::MemoryBufferLogger>(max_log_items_to_keep, prealloc_log_items));
 
     loggers.front()->info("DNF5 start");
+    std::string cmdline;
```

Comment:
If I am not mistaken method `std::string Context::get_cmdline()` format the same string.

Reply:
May be I am mistaken but following code might be redundant:
```
    for (int i = 0; i < argc; ++i) {
        if (i > 0) {
            cmdline += " ";
        }
        cmdline += argv[i];
    }
```
and could be replaced by `std::string cmdline = context.get_cmdline()`.

Reply:
The Context is not yet instantiated. But the Context does not need to have argv/argc attributes, so I pass only concatenated command line to it using `contect.set_cmdline()`. And context.get_cmdline() is now simplified to plain `return cmdline;`.

Reply:
LGTM, thank you